test/unit/utils/url.test.js

```diff
@@ -30,9 +32,11 @@ test('should strip port from hosts using http or https', function() {
   };
 
   url = Url.parseUrl('/domain/relative/url');
-  ok(!(/.*:80$/).test(url.host), ':80 is not appended to the host');
 
   document.createElement = origDocCreate;
+
+  ok(!(/.*:80$/).test(url.host), ':80 is not appended to the host');
```

Comment:
our overriding of createElement here broke qunit. So, I moved the `ok` to after we reset `createElement` back to the original value.